bindings/2.200/GeometryDash.bro

```diff
@@ -7326,7 +7326,7 @@ class LevelBrowserLayer : cocos2d::CCLayerColor, LevelManagerDelegate, FLAlertLa
 [[link(android)]]
 class SetTextPopupDelegate {
 	/* unverified signature */
-	virtual void setTextPopupClosed(SetTextPopup*, gd::string);
+	void setTextPopupClosed(SetTextPopup*, gd::string) {}
```

Comment:
why was the virtual removed? this is a delegate

Reply:
It was an attempt to try it on mac, I'll restore it.